### PR TITLE
Update phpunit.xml

### PR DESCRIPTION
**Pull request by MecOnLine**
-          backupStaticAttributes = " true "
+          backupStaticAttributes = " true "

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <phpunit backupGlobals="true"
-         backupStaticAttributes="true"
+         backupStaticAttributes="true"e
```
Comment (MecOnLine) [inline phpunit.xml]: +         backupStaticAttributes="true"

Comment (MecOnLine) [inline phpunit.xml]: backupStaticAttributes="true"

**Comment by barbushin** [comment]
All build failed

File: phpunit.xml (hunk 1 above)
Comment (MecOnLine) [inline phpunit.xml]: backupStaticAttributes="true"

**Pull request closed without merging**